### DISCUSSION TITLE
Task View instead of Timeline icon for Taskbar10

### DISCUSSION THREAD

**Discussion by Kichura** [valinet/ExplorerPatcher#2838] [Ideas]
Since Windows 11 has essentially cut the entire timeline feature to restore the task view design that was in Windows 10 build 9841 albeit redesigned to fit windows 11's aesthetics, having taskbar10 use the old task view icon instead of timeline would make a bit more sense in some contexts.

What Windows 10 RTM has:

![a](https://github.com/valinet/ExplorerPatcher/assets/68134602/df9f0ae8-8973-4f03-baec-9314b6bb9b36)

What Windows 10 22H2 has:

![b](https://github.com/valinet/ExplorerPatcher/assets/68134602/2b83bdd9-9697-4394-af38-239c5c8fe9ad)

**Comment by pyrates999** [comment]
Is this just an icon change?

**Reply by Kichura** [reply]
This wasn't only an icon change but also added the timeline feature to windows 10 when it was initially released but starting in windows 11, this feature was cut which is also the same reason why timeline changed back to task view there.

**Reply by pyrates999** [reply]
The timeline originally in windows 10 was requested as a new feature by someone else here and declined already.

**Comment by Amrsatrio** [comment]
Coming with the reimplemented taskbar.
<img width="722" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/20662640/67d14b36-1ea1-4092-b3cd-9e457dcdc00a">
